extensions/openshell-podman-gateway/src/helper/podman-info.ts

```
@@ -0,0 +1,177 @@
+  transientStore: boolean;
+}
+
+export interface ContainerStore {
+  number: number;
+  paused: number;
+  running: number;
+  stopped: number;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface GraphOptions {}
```

Biome (2.5.1)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)